### PR TITLE
.env files created

### PR DESCRIPTION
**Pull request by LGG233**
I think this is set - followed all the items in the checklist. I added a gitignore file in the server folder; you didn't include that in your list but seemed to make sense since the server has its own .env file.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const MongoStore = require('connect-mongo')(session)
 const passport = require('./passport');
 // route requires
 const user = require('./routes/user')
+require('dotenv').config()
```
Comment (kylewhitaker) [inline server/server.js:12]: Move to the very top of the file so that the `.env` file is the first thing loaded.

Comment (kylewhitaker) [inline server/server.js]: Move to the very top of the file so that the `.env` file is the first thing loaded.

File: server/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.env
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (kylewhitaker) [inline server/.gitignore]: This doesn't hurt; it allows the `server` to be even more of a standalone system. I support this! :smiley:

[nit] I'm a minimalist coder so I wonder if all of these things are needed? I realize this is a copy-paste from the `client`. Probably just `node_modules`, `.env` and (just for fun) `npm-debug.log` are needed.

**Review by kylewhitaker — APPROVED** [review]
Looks great! Nice job! Two little comments.

**Comment by kylewhitaker** [comment]
[.env files & docs](https://trello.com/c/4AREyrOL/1-env-files-docs)

File: server/.gitignore (hunk 2 above)
Comment (LGG233) [reply server/.gitignore]: OK, I made these changes. It looks like I can just push the changes without having to do another PR, but maybe that's not right? Anyway, that's what I did...

**Review by kylewhitaker — APPROVED** [review]

**Pull request merged**